pep-3156.txt

```diff
@@ -147,6 +148,15 @@ wait for a Future to complete by adding a callback to the Future.
 Likewise, the scheduler offers an operation to suspend a coroutine
 until a callback is called.
 
+In case of any other async framework cannot just use Future and Task
+as is for some reason the framework may reimplement
+``loop.create_future()`` and ``loop.create_task()`` calls by returning
+an object which implements a superset of Future/Task interfaces but
+adds a new functionality required by the framework.
+
+``loop.set_task_factory()`` may be useful as well for adopting asyncio
```

Comment:
Not really sure why we need to delete this. The function by itself is harmless, as you can't guard yourself from `Task` being overridden anyways (since you can subclass and shadow `create_task`). And we don't really encourage overriding loop functionality by subclassing -- all our other APIs are designed with that in mind.